pds/context.go

```diff
@@ -0,0 +1,60 @@
+package pds
+
+import (
+	"context"
+
+	"github.com/golang-jwt/jwt"
+)
+
+type (
+	ctxAuth      struct{}
+	ctxAuthScope struct{}
+	ctxDID       struct{}
+	ctxToken     struct{}
+	ctxUser      struct{}
```

Comment:
Use distinct types as keys for `context.WithValue`, not strings, to avoid collisions with other packages doing the same thing.

This file encapsulates setting and getting of the particular context values needed by this package.